docs/tutorials/average_optimizers_callback.ipynb

```diff
@@ -0,0 +1,636 @@
+            "Epoch 00002: saving model to ./training\n",
+            "1875/1875 [==============================] - 4s 2ms/step - loss: 0.4944 - accuracy: 0.8291\n",
+            "Epoch 3/5\n",
+            "1873/1875 [============================>.] - ETA: 0s - loss: 0.4485 - accuracy: 0.8429\n",
+            "Epoch 00003: saving model to ./training\n",
+            "1875/1875 [==============================] - 4s 2ms/step - loss: 0.4483 - accuracy: 0.8430\n",
+            "Epoch 4/5\n",
+            "1869/1875 [============================>.] - ETA: 0s - loss: 0.4233 - accuracy: 0.8519\n",
+            "Epoch 00004: saving model to ./training\n",
+            "1875/1875 [==============================] - 4s 2ms/step - loss: 0.4233 - accuracy: 0.8519\n",
+            "Epoch 5/5\n",
+            "1868/1875 [============================>.] - ETA: 0s - loss: 0.4020 - accuracy: 0.8590\n",
+            "Epoch 00005: saving model to ./training\n",
+            "1875/1875 [==============================] - 4s 2ms/step - loss: 0.4020 - accuracy: 0.8591\n"
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "<tensorflow.python.keras.callbacks.History at 0x7fab49765be0>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 10
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "zRAym9EBmnW9",
+        "colab_type": "code",
+        "outputId": "42713d82-c2ec-4f7a-8375-07560c7a5733",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 69
+        }
+      },
+      "source": [
+        "#Evalute results\n",
+        "model.load_weights(checkpoint_dir)\n",
+        "loss, accuracy = model.evaluate(test_images, test_labels, batch_size=32, verbose=2)\n",
+        "print(\"Loss :\", loss)\n",
+        "print(\"Accuracy :\", accuracy)"
+      ],
+      "execution_count": 11,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "10000/10000 - 0s - loss: 86.9890 - accuracy: 0.7955\n",
+            "Loss : 86.98903251190185\n",
+            "Accuracy : 0.7955\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "K98lbU07m_Bk",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Stocastic Weight Average SGD "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Ia7ALKefnXWQ",
+        "colab_type": "code",
+        "outputId": "97fb462e-2c08-41d5-956d-a2ddb5140ee9",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 399
+        }
+      },
+      "source": [
+        "#Build Model\n",
+        "model.load_weights(checkpoint_dir)\n",
```

Comment:
This line does not seem to be useful.

Reply:
Yes it's redundant. I think I'll switch to the colab workflow for better diffs.